packages/twenty-front/src/modules/object-metadata/components/ObjectMetadataNavItems.tsx

```diff
@@ -17,6 +17,11 @@ import { useNavigationSection } from '@/ui/navigation/navigation-drawer/hooks/us
 import { View } from '@/views/types/View';
 import { getObjectMetadataItemViews } from '@/views/utils/getObjectMetadataItemViews';
 
+const navItemsAnimationVariants = {
+  hidden: { height: 0 },
+  visible: { height: 'auto' },
+};
```

Comment:
Hello @AliElamir, thank you for contributing.
The animation is not exactly what @Bonapara described in his issue.
When you display your animation slowly, you can see that the children appear instantly and then the parent element height is animated, I feel like you should also animate the height of the children elements to make it feel smoother.

https://github.com/user-attachments/assets/7f81bc90-10ce-4836-92c2-05d2d6684909

More insights:
- the animation should also be displayed when we close the menu
- the opacity also seems to be animated from 0 to 1

Thank you :)

Reply:
Hey @bosiraphael , thank you for your feedback. 

I have adjusted the animation based on your comments and pushed the latest changes. It looks right to me now, but please let me know if there are any more adjustments needed. @Bonapara 

Thank you!